plugins/file/plugin.go

```diff
@@ -77,7 +77,36 @@ func Handler6(req, resp dhcpv6.DHCPv6) (dhcpv6.DHCPv6, bool) {
 		return nil, false
 	}
 	log.Printf("Found IP address %s for MAC %s", ipaddr, mac)
-	// TODO add an OptIANA based on the above data
+	resp.AddOption(&dhcpv6.OptIANA{
+		// FIXME this must be unique per client address
+		IaId: [4]byte{0xaa, 0xbb, 0xcc, 0xdd},
+		T1:   0,
```

Comment:
Nit: you can technically omit these because they're zero value.

Reply:
right! I originally set it to 3600 because I confused it with ValidLifetime and PreferredLifetime, and didn't get rid of it when reverted

Reply:
done